OneCommunityGlobal/HighestGoodNetworkApp
# Aaron eslint fixes for Timelog component

# Description
Fix linting errors for the Timelog component

## Related PRS (if any):
N/A

## Main changes explained:
- removed the Timelog component directory from `.eslintignore` and `.prettierignore`
- fixed lint errors in Timelog component directory

## How to test:
1. check into current branch
2. do `npm install` and `npm run lint` to make sure you get no lint errors
3. run `npm run start:local` and start the backend as usual
4. make sure the app works as usual (make sure you can log time as normal)
5. run `npm run test` to make sure the tests are passing (see below image)

![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/10265513/6fbdde37-d359-412b-8f01-ec5574d4d1e1)

Note: the failing tests are also on dev branch, I am looking into that
